eventually/Cargo.toml

```diff
@@ -12,19 +12,16 @@ categories = ["rust-patterns", "web-programming", "asynchronous", "data-structur
 keywords = ["architecture", "ddd", "event-sourcing", "cqrs", "es"]
 
 [features]
-default = []
-serde = ["eventually-core/serde"]
-# TODO: is there a way to exclude eventually-postgres/with-tracing if postgres
-# hasn't been specified in features?
-tracing = ["eventually-core/with-tracing", "eventually-util/with-tracing", "eventually-postgres/with-tracing"]
-postgres = ["eventually-postgres"]
-full = ["serde", "tracing", "postgres"]
 
 [dependencies]
-eventually-core = { version = "0.4.0", path = "../eventually-core" }
-eventually-util = { version = "0.4.0", path = "../eventually-util" }
-eventually-postgres = { version = "0.3.0", path = "../eventually-postgres", optional = true }
+anyhow = "1.0.38"
```

Comment:
it's a little unusual to use `anyhow` in a library- normally you'd prefer `thiserror` and reserve `anyhow` for binary crates. Can you explain the design choice? (i'm perfectly prepared to be convinced there's a good reason!)